.markdownlint.yaml

```diff
@@ -11,4 +11,6 @@ no-duplicate-header:
 ol-prefix:
   style: ordered
 no-inline-html: false
-fenced-code-language: false
\ No newline at end of file
+fenced-code-language: false
+MD024: false
+MD059: false
```

Comment:
Do these need to be ignored if using the most recent version? Is the bug fixed there?